modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala

```diff
@@ -177,6 +173,38 @@ object Java {
       }
   }
 
+  def sortDefinitions(defns: List[BodyDeclaration[_]]): List[BodyDeclaration[_]] = {
+    import com.github.javaparser.ast.Modifier._
+    def sortKeyFor(x: BodyDeclaration[_]): Int = x match {
+      case cd: ClassOrInterfaceDeclaration if cd.getModifiers.contains(PUBLIC)                              => 0
+      case cd: ClassOrInterfaceDeclaration if !cd.getModifiers.contains(PRIVATE)                            => 10
```

Comment:
`if (a) {} else if (!a) {} else {}`?

Reply:
This comment does make sense for a few of the cases below, but not this one.  Will clean up the other ones before merging.

Reply:
(Remember, the order in Java for access is public > protected > (no modifier) > private.)

Reply:
Honestly misread this to be `PUBLIC` `!PUBLIC`